main.go

```diff
@@ -2,13 +2,28 @@ package main
 
 import (
 	"net/http"
+	"os"
 
+	notify "github.com/alphagov/notifications-go-client"
 	"github.com/gorilla/mux"
 	"github.com/onsdigital/log.go/log"
 )
 
+var client notify.Client
+
 func main() {
 	log.Event(nil, "Starting rm-notify-gateway")
+
+	config := notify.Configuration{
+		APIKey:    []byte(os.Getenv("NOTIFY_API_KEY")),
+		ServiceID: os.Getenv("NOTIFY_SERVICE_ID"),
+	}
+
+	client, err := notify.New(config)
+	if err != nil {
+		panic(1)
```

Comment:
This is me being a little bit lazy; it really needs to be properly logged, but `log.go` is really unfriendly to use; I'm likely to take it out in future work